.github/workflows/ci.yml

```
@@ -220,20 +219,26 @@
           
           # Create tags and releases only for production deployments
           if [ "$DEPLOYMENT_TYPE" = "production" ]; then
+            # Always create git tag for Sentry tracking
+            git tag "v${NEW_VERSION}"
+            git push origin "v${NEW_VERSION}"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)